doc/release/platforms/infiniband.rst

```diff
@@ -0,0 +1,177 @@
+.. _readme-infiniband:
```

Comment:
It might be worthwhile to have a link here from the CS section in platforms/cray.